### ISSUE TITLE
Chat component

### ISSUE THREAD

**Issue by rcarmo** [FlowFuse/node-red-dashboard#539]
### Description

Given the industry-wide focus on AI and the fact that we now have personalised dashboards, it might be interesting to add a chat component:

https://github.com/advanced-chat/vue-advanced-chat

This would be useful for:

- Talking to LLMs
- Chatting between different users accessing the same dashboard

### Have you provided an initial effort estimate for this issue?

I am no FlowFuse team member

**Comment by joepavitt** [comment]
@rcarmo appreciate we're late to this, but just an update that we will be going ahead with building this, albeit as a standalone widget for installation, not part of the Dashboard core collection. I'll update this issue and link to new repositories, etc. once they're available.

**Comment by rcarmo** [comment]
Thanks. I am still waiting on #586 before I can consider moving our deployments to the new dashboard, but appreciate the update.
